include/sbi/sbi_scratch.h

```diff
@@ -37,7 +37,8 @@
 /** Offset of ipi_type in sbi_ipi_data */
 #define SBI_IPI_DATA_IPI_TYPE_OFFSET		(15 * __SIZEOF_POINTER__)
 
-#define SBI_SCRATCH_TLB_INFO_OFFSET		(16 * __SIZEOF_POINTER__)
+#define SBI_SCRATCH_TLB_INFOQ_OFFSET		(16 * __SIZEOF_POINTER__)
+
```

Comment:
This would create 32 bytes of gap in scratch. How about just keeping queue right after where the header ends.

#define SBI_SCRATCH_FIFO_HEADER_SIZE sizeof(struct sbi_fifo)
#define SBI_SCRATCH_TLB_QUEUE_MEM_OFFSET (SBI_SCRATCH_TLB_QUEUE_HEAD_OFFSET + SBI_SCRATCH_FIFO_HEADER_SIZE)

We don't have to assign queue memory by hand as well.